src/Features/VisualBasic/Portable/Structure/VisualBasicBlockStructureProvider.vb

```diff
@@ -88,6 +88,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Structure
 
             builder.Add(SyntaxKind.DisabledTextTrivia, ImmutableArray.Create(Of AbstractSyntaxStructureProvider)(New DisabledTextTriviaStructureProvider()))
 
+            builder.Add(SyntaxKind.CommentTrivia, ImmutableArray.Create(Of AbstractSyntaxStructureProvider)(New CommentTriviaStructureProvider()))
```

Comment:
what is this line for?

Reply:
The comment outlining provider is new provider.